### PR TITLE
Remove storyquestios

### PR DESCRIPTION
**Pull request by regiskuckaertz**
and upgrade JS to get rid of all those security alerts 😱

### CODE REVIEW

File: core/src/main/resources/commonsdivision/article/index.scala.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import com.gu.contentatom.renderer.utils.GuardianDateFormatter
 @import com.gu.contentatom.thrift.atom.commonsdivision.Mp
+@import scala.collection.immutable.Map
+@import scala.collection.immutable.Map
```
Comment (JustinPinner) [inline core/src/main/resources/commonsdivision/article/index.scala.html]: These look the same to me. Am I missing something 🤔

File: package.json [hunk 2]
```diff
@@ -1,54 +1,54 @@
 {
   "name": "@guardian/atom-renderer",
-  "version": "1.0.4",
+  "version": "1.17.3",
```
Comment (JustinPinner) [inline package.json]: Blimey, that's a leap!

File: core/src/main/resources/commonsdivision/article/index.scala.html [hunk 3]
```diff
@@ -104,7 +106,7 @@ <h4 class="atom--snippet__headline">'@Html(title)'</h4>
     </summary>
 
     <div class="atom--snippet__body atom--commonsdivision__body">
-      @{partiesTable(data.votes.ayes.groupBy(_.party), data.votes.noes.groupBy(_.party))}
+      @{partiesTable(data.votes.ayes.toSeq.groupBy(_.party), data.votes.noes.toSeq.groupBy(_.party))}
```
Comment (JustinPinner) [inline core/src/main/resources/commonsdivision/article/index.scala.html]: Can you tell me how is this related to removing storyquestions?

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala [hunk 4]
```diff
@@ -44,9 +44,9 @@ object Main extends IOApp {
 
   val client = new GuardianContentClient(sys.env("CAPI_TEST_KEY"))
 
-  val assetsService: HttpRoutes[IO] = HttpRoutes.of[IO] {
+  def assetsService(blocker: Blocker): HttpRoutes[IO] = HttpRoutes.of[IO] {
```
Comment (JustinPinner) [inline utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: Again, how is this related to storyquestions? I'll have to look up what Blocker is and what it does.

Comment (JustinPinner) [reply utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: Ok, so I assume the `global` execution context was made available by `import org.http4s.implicits._`, and http4s `v0.21.0-M1` introduced a breaking change where `ExecutionContext` was replaced with "cats-effect-2.0’s new `Blocker`". Nothing at all obvious about any of that.

File: core/src/main/resources/commonsdivision/article/index.scala.html (hunk 1 above)
Comment (regiskuckaertz) [reply core/src/main/resources/commonsdivision/article/index.scala.html]: woops

File: package.json (hunk 2 above)
Comment (regiskuckaertz) [reply package.json]: woopsie too

File: core/src/main/resources/commonsdivision/article/index.scala.html (hunk 3 above)
Comment (regiskuckaertz) [reply core/src/main/resources/commonsdivision/article/index.scala.html]: I upgraded to scala 2.13 at the same time ✌️

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala (hunk 4 above)
Comment (regiskuckaertz) [reply utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: yeah , this is def not clear , and i'm kind of sick of the cats-effect project at this stage. when i've got time i'll replace that with something a bit more usable.

**Comment by regiskuckaertz** [comment]
Thank you.
